src/swipes.py

```diff
@@ -0,0 +1,304 @@
+from os.path import isfile
+from datetime import date, datetime, timedelta
+import json
+import numpy as np
+import pandas as pd
+
+from src.constants import (
+  BRB_ONLY,
+  DINING_HALL,
+  EATERY_DATA_PATH,
+  LOCATION_NAMES,
+  ISOLATE_COUNTER_SWIPES,
+  ISOLATE_DATE,
+  ISOLATE_SWIPES,
+  SCHOOL_BREAKS,
+  TRILLIUM,
+  WAIT_TIME_CONVERSION,
+  WEEKDAYS,
+)
+from src.eatery import string_to_date_range
+from src.types import SwipeDataType
+
+weekdays = {index: day for day, index in WEEKDAYS.items()}  # invert our weekday -> index dict
```

Comment:
might be more readable & cleaner to do:

`{v:k for k, v in ...}`, just to make it EXPLICIT that the keys & values are being inverted, and the comment should explain why this is happening